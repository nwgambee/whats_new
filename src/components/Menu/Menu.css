* {
  font-family: 'Raleway', sans-serif;
}

button {
  background-color: #C04ABC;
  font-size: 1em;
  font-weight: 700;
  border-radius: 7px;
  padding: 4px;
  margin: 5px;
  border: none;
  box-shadow: 1px 1px;
}
