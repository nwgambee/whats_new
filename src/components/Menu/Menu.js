import React from 'react';
import './Menu.css'

const Menu = ({changeSource}) => {
  return (
    <div className='Menu'>
      <button className='news-src-btn' onClick={changeSource} name='entertainment'>Entertainment</button>
      <button className='news-src-btn' onClick={changeSource} name='health'>Health</button>
      <button className='news-src-btn' onClick={changeSource} name='local'>Local</button>
      <button className='news-src-btn' onClick={changeSource} name='science'>Science</button>
      <button className='news-src-btn' onClick={changeSource} name='technology'>Technology</button>
    </div>
  )
}

export default Menu;
