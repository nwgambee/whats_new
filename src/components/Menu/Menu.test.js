import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Menu from './Menu';

describe('Menu', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Menu
      onClick={jest.fn()}
            />);

    expect(wrapper).toMatchSnapshot();
  });
});
