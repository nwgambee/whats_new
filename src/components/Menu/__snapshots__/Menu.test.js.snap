// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should match the snapshot 1`] = `
<div
  className="Menu"
>
  <button
    className="news-src-btn"
    name="entertainment"
  >
    Entertainment
  </button>
  <button
    className="news-src-btn"
    name="health"
  >
    Health
  </button>
  <button
    className="news-src-btn"
    name="local"
  >
    Local
  </button>
  <button
    className="news-src-btn"
    name="science"
  >
    Science
  </button>
  <button
    className="news-src-btn"
    name="technology"
  >
    Technology
  </button>
</div>
`;
